### PR TITLE
NO-JIRA:  Enable skip mco reboot functionality only if openshift version is 4.15 and higher

### PR DESCRIPTION
**Pull request by ori-amizur**
## List all the issues related to this PR

- [ ] New Feature 
- [x] Enhancement 
- [ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [x] Cloud
- [x] Operator Managed Deployments
- [ ] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [ ] No tests needed

## Checklist

- [ ] Title and description added to both, commit and PR.
- [ ] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [ ] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- Are the title and description (in both PR and commit) meaningful and clear?
- Is there a bug required (and linked) for this change?
- Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md
/cc @tsorya

### CODE REVIEW

File: internal/host/hostcommands/install_cmd.go [hunk 1]
```diff
@@ -24,6 +24,8 @@ import (
 	"gorm.io/gorm"
 )
 
+const baseVersionForSkipMcoReboot = "4.15.0"
```
Comment (tsorya) [inline internal/host/hostcommands/install_cmd.go:27]: We have feature list per version maybe better to add it there?

Comment (tsorya) [reply internal/host/hostcommands/install_cmd.go:27]: featuresupport package i think

**Comment by gamli75** [comment]
/pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-4-15

**Comment by gamli75** [comment]
pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-4-15

**Comment by danmanor** [comment]
Do we want to wait for `edge-e2e-metal-assisted-4-15`  ? https://github.com/openshift/release/pull/45505

**Pull request merged**